otter/log/formatters.py

```diff
@@ -237,6 +239,22 @@ def audit_log_formatter(eventDict, timestamp, hostname):
     return audit_log_params
 
 
+@singledispatch
+def serialize_to_jsonable(obj):
+    """
+    Serialize any object to a JSONable form
+    """
+    return None
```

Comment:
So, I thought of that but left it due to formatter [checking](https://github.com/rackerlabs/otter/pull/1197/files#diff-193b770a377724dd1647e589b8022d0dR283) it. Since exception message is already taken care I was not sure if it was a good idea to add it in `error_details` also.